modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/classic_richardson.cc

```diff
@@ -0,0 +1,136 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#include "modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/classic_richardson.h"
+#include "modules/linear_boltzmann_solvers/discrete_ordinates_solver/iterative_methods/sweep_wgs_context.h"
+#include "modules/linear_boltzmann_solvers/lbs_solver/acceleration/diffusion_mip_solver.h"
+#include "modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.h"
+#include "framework/math/linear_solver/linear_solver.h"
+#include "framework/logging/log.h"
+#include "framework/utils/timer.h"
+#include "framework/runtime.h"
+#include <memory>
+#include <iomanip>
+
+namespace opensn
+{
+namespace lbs
+{
```

Comment:
🗒️  If #328 goes in before this PR, then this will need to be removed before merging

Reply:
Just a comment that #328 is in, so do not forget to remove this before merging...